.github/workflows/test-env-deploy.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         id: set-domain
         # Set test instance domain based on branch name slug
         run: |
-          echo "::set-output name=domain::${{ env.GITHUB_HEAD_REF_SLUG_URL }}.dashboard.saleor.rocks"
```

Comment:
It should not be modified. Test environments are hosted outside cloud, under saleor.rocks domain.